integration-tests/hibernate-orm-tenancy/connection-resolver-legacy-qualifiers/src/main/resources/application.properties

```diff
@@ -3,7 +3,7 @@ quarkus.hibernate-orm.database.generation=none
 quarkus.hibernate-orm.multitenant=database
 # Necessary because we're creating datasources dynamically,
 # which means the extension can't rely on a static datasource to guess the dialect
-quarkus.hibernate-orm.dialect=org.hibernate.dialect.MariaDBDialect
+quarkus.hibernate-orm.dialect=MariaDB
```

Comment:
How about updating the `quarkus.hibernate-orm."inventory".dialect` or did you keep it an FQCN so that we test both scenarios?